src/Notepads/Notepads.csproj

```diff
@@ -860,6 +1016,10 @@
     <VisualStudioVersion>14.0</VisualStudioVersion>
   </PropertyGroup>
   <Import Project="$(MSBuildExtensionsPath)\Microsoft\WindowsXaml\v$(VisualStudioVersion)\Microsoft.Windows.UI.Xaml.CSharp.targets" />
+  <PropertyGroup Condition="'$(Configuration)' == 'Production'">
+    <PreBuildEvent>powershell -ExecutionPolicy Bypass $(ProjectDir)build.ps1 -project_dir $(ProjectDir) -target PreBuild</PreBuildEvent>
```

Comment:
Why it has both Pre and Post-build under Production section?

Reply:
Pre-build event modifies the manifest to be used for Store release, post-build event restores the manifest to default.